src/styl/now-value.styl

```diff
@@ -0,0 +1,22 @@
+@import '_variables'
+
+.dollert-now-value-container
+  padding-bottom: 15px
+  text-align: center
+  line-height: 20px
+  background $color-white
```

Comment:
It's inherting `background-color: $color-grey-light` from the popup.styl since the `dollar-now-value-container` is outside the form (I don't think it should be inside, since has nothing to do with the form).